src/py/flwr/server/grpc_server/certificates/certificate.conf

```diff
@@ -0,0 +1,20 @@
+[req]
+default_bits = 4096
+prompt = no
+default_md = sha256
+req_extensions = req_ext
+distinguished_name = dn
```

Comment:
Maybe:
```suggestion
distinguished_name = @dn
```

Reply:
@danieljanes this is correct as described in here: https://www.openssl.org/docs/manmaster/man5/x509v3_config.html#Certificate-Policies the @syntax is only used in some cases e.g. when the values as far as I understood could also be inline.